apps/spruce/src/pages/waterfall/TaskStatsTooltip.tsx

```diff
@@ -27,7 +27,7 @@ export const TaskStatsTooltip: React.FC<
     0;
 
   return (
-    <>
+    <div>
```

Comment:
Adding this div fixes above where the rightmost task stats popover was not positioned correctly